<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmationDialog</class>
 <widget class="QDialog" name="ConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="5,1" rowminimumheight="0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnYes">
       <property name="text">
        <string>yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNo">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
